appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleTracker.kt

```diff
@@ -54,6 +57,7 @@ internal class ExperienceLifecycleTracker(
                         trackLifecycleEvent(StepSeen(it.experience, it.flatStepIndex))
                     }
                     is EndingStep -> {
+                        trackFormSubmission(it.experience, it.flatStepIndex)
```

Comment:
this is where we'll need more conditional logic in the future to determine if and when form results should be submitted - right now its just on every ending step transition.